### DISCUSSION TITLE
Can't seem to update

### DISCUSSION THREAD

**Discussion by PerpSearch** [benjaminjonard/koillection#1249] [Q&A] [answered]
Normally this has not been a problem, but w/ the last release I can't seem to get to v 1.5.15.   I saw the PHP 8.3 is required part, but if I'm running 8.3.12 I should be OK.  
![2024-11-25_192134](https://github.com/user-attachments/assets/f11a777a-d7fb-47f9-bd79-badf213728c7)

**Comment by benjaminjonard** [comment]
Hello, how do you update Koillection ? If you use Docker could you please copy paste your docker compose file ?

**Comment by PerpSearch** [comment]
Normally I don't  have to do anything as I use Watchtower which automatically checks for updates and once found does it all for me.  It's worked through every update since 04/2022.  Still works, but I like to stay current. I notice the tag says koillection/koillection which as worked, but I when I changed it to koillection/latest it errored out.  I did redeploy it a few times before I posted the issue and I notice the present distro is dated 2024-10-28 07:33:20.  It's probably something simple. Anyway, this is my compose file and thanks:

version: '3.9'
services:
    koillection:
        image: koillection/koillection
        container_name: Koillection
        restart: on-failure:5
        ports:
            - 4460:80
        environment:
            - DB_DRIVER=pdo_pgsql
            - DB_NAME=koillection
            - DB_HOST=db
            - DB_PORT=5432
            - DB_USER=koillectionuser
            - DB_PASSWORD=koillectionpass
            - DB_VERSION=16
            - PHP_TZ=America/Detroit
            - HTTPS_ENABLED=0
            - PUID=1026
            - PGID=100
        depends_on:
            - db
        volumes:
            - /volume1/docker/koillection/conf:/conf:rw
            - /volume1/docker/koillection/uploads:/uploads:rw
    db:
        image: postgres:16
        container_name: Koillection-DB
        mem_limit: 512m
        cpu_shares: 768
        security_opt:
            - no-new-privileges:true
        healthcheck:
         test: ["CMD", "pg_isready", "-q", "-d", "koillection", "-U", "koillectionuser"]
         timeout: 45s
         interval: 10s
         retries: 10
        environment:
            - POSTGRES_DB=koillection
            - POSTGRES_USER=koillectionuser
            - POSTGRES_PASSWORD=koillectionpass
        volumes:
            - /volume1/docker/koillection/db:/var/lib/postgresql/data:rw
        restart: on-failure:5

**Reply by PerpSearch** [reply]
Also, here's the log

Logo

Containers>Koillection>Logs
Container logs
Log viewer settings
Auto-refresh logs
Wrap lines
Display timestamps
Fetch
Search
Lines
Actions

      

[notice] Migration App\Migrations\Mysql\Version20240105142108 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20240119000437 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20240204172429 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20240205152905 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20240329230249 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20240411083658 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20240906163226 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] finished in 42.1ms, used 14M memory, 44 migrations executed, 0 sql queries

 [OK] Successfully migrated to version:                                         

      App\Migrations\Postgresql\Version20240906153414                           

**** 5/11 - Create API keys ****

 // Your key files already exist, they won't be overriden.                      

**** 6/11 - Refresh caches ****

Refreshing cached values for collections...

Refreshing cached values for wishlists...

Refreshing cached values for albums...

Done!

**** 7/11 - Create user and use PUID/PGID ****

-e  	User UID :	1026

-e  	User GID :	100

**** 8/11 - Set Permissions ****

**** 9/11 - Create nginx log files ****

**** 10/11 - Create symfony log files ****

**** 11/11 - Setup complete, starting the server. ****

**** 1/11 - Make sure /uploads folders exist ****

**** 2/11 - Create the symbolic link for the /uploads folder ****

**** 3/11 - Setting env variables ****

**** 4/11 - Migrate the database ****

[notice] Migrating up to App\Migrations\Postgresql\Version20240906153414

[notice] Migration App\Migrations\Mysql\Version20200429093452 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20200727114843 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20200812160901 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20200923193955 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20201010121132 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20201215214207 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20210127105342 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20210514100228 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20211207185256 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20220206133834 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20220206175335 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20220208094335 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20220630100634 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20220701175209 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20220713152238 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20220716201458 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20220717140437 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20220722163001 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20220722192217 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20220725204054 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20220829085551 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20220909124604 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20220916162932 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20220917151134 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20220918183100 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20220920150306 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20220922183351 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20230317131927 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20230425131232 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20230519201736 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20230525100104 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20230613175623 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20230726074848 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20230801111844 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20230801130420 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20230803153034 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20231124105513 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20240105142108 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20240119000437 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20240204172429 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20240205152905 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20240329230249 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20240411083658 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] Migration App\Migrations\Mysql\Version20240906163226 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."

[notice] finished in 274ms, used 14M memory, 44 migrations executed, 0 sql queries

 [OK] Successfully migrated to version:                                         

      App\Migrations\Postgresql\Version20240906153414                           

**** 5/11 - Create API keys ****

 // Your key files already exist, they won't be overriden.                      

**** 6/11 - Refresh caches ****

Refreshing cached values for collections...

Refreshing cached values for wishlists...

Refreshing cached values for albums...

Done!

**** 7/11 - Create user and use PUID/PGID ****

-e  	User UID :	1026

-e  	User GID :	100

**** 8/11 - Set Permissions ****

**** 9/11 - Create nginx log files ****

**** 10/11 - Create symfony log files ****

**** 11/11 - Setup complete, starting the server. ****

**Comment by benjaminjonard** [comment]
For some reasons the docker image build process failed without notifying me, so the 1.5.15 tag wasn't available. I rebuilt it, you should be able to update now

**Reply by PerpSearch** [reply]
I did but, as you probably know by now things are not going as expected today.  Not sure if this is useful, but it's what I have.  Also, I tried to restore a version from 5 days ago and that didn't seem to improve things. I can send the new logs, but it looks like you have more info than you need ATM.  



![2024-11-29_123753](https://github.com/user-attachments/assets/5c9650bd-fb15-4882-a7b5-b77ec5359af0)

**Comment by benjaminjonard** [comment]
I'll continue the discussion here so we don't spam others with notifications

You kept your volumes folders somewhere right ? Just starting your docker containers with the folders at the right place should be enough

**Reply by PerpSearch** [reply]
Thanks. I have about 4 days backed up and a copy as well. I started a new clean install and that worked now I have to get the data back. Since I used the same info username PW email etc, I thought I could just relocate the original data folders to their original positions, but that doesn't seem to have worked yet. I'll try restarting the containers now and we'll see. BRB

**Reply by PerpSearch** [reply, marked as answer]
Creating a new install, replacing the original folders with the original data, and restarting the containers worked. The only thing that worked.  I thought I was screwed for sure.  Thanks and I'll be increasing the number of backups just in case.

**Comment by benjaminjonard** [comment]
Trying to answer your latest issue here, GitHub is buggy and won't let me post new comments on issues...

Could you try to add the env variable APP_DEBUG=1 and restart the container? It should display a more detailed error

**Comment by PerpSearch** [comment]
While I was waiting I tried what worked last time and renamed the
koillection docker folder to backup and created a new koillection docker
folder and subs
I redeployed the stack and it works w/ zero records.  I then replaced the
original docker folder and restarted the container and still zero records.
I've now added the APP_DEBUG=1
and of course it's not working. The log is attached. I'll try to check
GitHub aslo.  Thanks for looking into this.
C

On Tue, Dec 3, 2024 at 2:46 PM Benjamin Jonard ***@***.***>
wrote:

> Trying to answer your latest issue here, GitHub is buggy and won't let me
> post new comments on issues...
>
> Could you try to add the env variable APP_DEBUG=1 and restart the
> container? It should display a more detailed error
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/benjaminjonard/koillection/discussions/1249#discussioncomment-11452464>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AV6KCRZFQOUKRZAOUXU4THT2DYC77AVCNFSM6AAAAABSPGHE56VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNBVGI2DMNA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> .com>
>

**** 1/11 - Make sure /uploads folders exist ****
**** 2/11 - Create the symbolic link for the /uploads folder ****
**** 3/11 - Setting env variables ****
**** 4/11 - Migrate the database ****
[2024-12-03T20:30:15.395153+00:00] deprecation.INFO: Deprecated: Lexik\Bundle\JWTAuthenticationBundle\Services\JWSProvider\LcobucciJWSProvider::__construct(): Implicitly marking parameter $clock as nullable is deprecated, the explicit nullable type must be used instead {"exception":"[object] (ErrorException(code: 0): Deprecated: Lexik\\Bundle\\JWTAuthenticationBundle\\Services\\JWSProvider\\LcobucciJWSProvider::__construct(): Implicitly marking parameter $clock as nullable is deprecated, the explicit nullable type must be used instead at /var/www/koillection/vendor/lexik/jwt-authentication-bundle/Services/JWSProvider/LcobucciJWSProvider.php:47)"} []
[2024-12-03T20:30:15.413600+00:00] deprecation.INFO: Deprecated: Lexik\Bundle\JWTAuthenticationBundle\Services\JWTManager::__construct(): Implicitly marking parameter $payloadEnrichment as nullable is deprecated, the explicit nullable type must be used instead {"exception":"[object] (ErrorException(code: 0): Deprecated: Lexik\\Bundle\\JWTAuthenticationBundle\\Services\\JWTManager::__construct(): Implicitly marking parameter $payloadEnrichment as nullable is deprecated, the explicit nullable type must be used instead at /var/www/koillection/vendor/lexik/jwt-authentication-bundle/Services/JWTManager.php:33)"} []
[notice] Migrating up to App\Migrations\Postgresql\Version20241029150732
[notice] Migration App\Migrations\Mysql\Version20200429093452 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20200727114843 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20200812160901 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20200923193955 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20201010121132 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20201215214207 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20210127105342 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20210514100228 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20211207185256 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20220206133834 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20220206175335 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20220208094335 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20220630100634 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20220701175209 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20220713152238 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20220716201458 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20220717140437 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20220722163001 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20220722192217 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20220725204054 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20220829085551 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20220909124604 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20220916162932 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20220917151134 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20220918183100 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20220920150306 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20220922183351 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20230317131927 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20230425131232 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20230519201736 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20230525100104 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20230613175623 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20230726074848 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20230801111844 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20230801130420 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20230803153034 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20231124105513 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20240105142108 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20240119000437 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20240204172429 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20240205152905 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20240329230249 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20240411083658 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20240906163226 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] Migration App\Migrations\Mysql\Version20241029164924 skipped during Execution. Reason: "Migration can only be executed safely on 'mysql' or 'mariadb'."
[notice] finished in 76.4ms, used 60.5M memory, 45 migrations executed, 0 sql queries
 [OK] Successfully migrated to version:                                         
      App\Migrations\Postgresql\Version20241029150732                           
**** 5/11 - Create API keys ****
 [OK] Done!                                                                     
**** 6/11 - Refresh caches ****
Refreshing cached values for collections...
Refreshing cached values for wishlists...
Refreshing cached values for albums...
Done!
**** 7/11 - Create user and use PUID/PGID ****
-e  	User UID :	1026
-e  	User GID :	100
**** 8/11 - Set Permissions ****
**** 9/11 - Create nginx log files ****
**** 10/11 - Create symfony log files ****
**** 11/11 - Setup complete, starting the server. ****

**Comment by PerpSearch** [comment]
I tried the 'fix' from last time where I did the install, got it working,
then replaced my original docker folders to the appropriate location, but
this time they fail. I get exit code 7 on the server and 1 on the db.
I've tried this with ver 1.15.14 and the results were about the same, but I
can't vouch for the reason.  I'm hoping that my data is corrupted.  I'm
going to try to restore from an earlier version now, but I've been trying
different things for about 4 hours now.

**Comment by PerpSearch** [comment]
and this is the latest log output from starting with a fresh install and my
original data, but I used a new stack provided by mariushosting.com dated
12/02.  This is my compose and the log will be attached

version: '3.9'
services:
    koillection:
        image: koillection/koillection
        container_name: Koillection
        restart: on-failure:5
        ports:
            - 4460:80
        environment:
            - APP_DEBUG=1
            - DB_DRIVER=pdo_pgsql
            - DB_NAME=koillection
            - DB_HOST=db
            - DB_PORT=5432
            - DB_USER=koillectionuser
            - DB_PASSWORD=koillectionpass
            - DB_VERSION=16
            - PHP_TZ=America/Detroit
            - HTTPS_ENABLED=0
            - PUID=1026
            - PGID=100
        depends_on:
            - db
        volumes:
            - /volume1/docker/koillection/conf:/conf:rw
            - /volume1/docker/koillection/uploads:/uploads:rw
    db:
        image: postgres:16
        container_name: Koillection-DB
        mem_limit: 512m
        cpu_shares: 768
        security_opt:
            - no-new-privileges:true
        healthcheck:
         test: ["CMD", "pg_isready", "-q", "-d", "koillection", "-U",
"koillectionuser"]
         timeout: 45s
         interval: 10s
         retries: 10
        environment:
            - POSTGRES_DB=koillection
            - POSTGRES_USER=koillectionuser
            - POSTGRES_PASSWORD=koillectionpass
        volumes:
            - /volume1/docker/koillection/db:/var/lib/postgresql/data:rw
        restart: on-failure:5


On Tue, Dec 3, 2024 at 2:46 PM Benjamin Jonard ***@***.***>
wrote:

> Trying to answer your latest issue here, GitHub is buggy and won't let me
> post new comments on issues...
>
> Could you try to add the env variable APP_DEBUG=1 and restart the
> container? It should display a more detailed error
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/benjaminjonard/koillection/discussions/1249#discussioncomment-11452464>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AV6KCRZFQOUKRZAOUXU4THT2DYC77AVCNFSM6AAAAABSPGHE56VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNBVGI2DMNA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> .com>
>

  An exception occurred in the driver: SQLSTATE[08006] [7] connection to serv  
  er at "db" (172.31.0.3), port 5432 failed: FATAL:  the database system is s  
  tarting up                                                                   
                                                                               
In Exception.php line 28:
                                                                               
  SQLSTATE[08006] [7] connection to server at "db" (172.31.0.3), port 5432 fa  
  iled: FATAL:  the database system is starting up                             
                                                                               
In PDOConnect.php line 26:
                                                                               
  SQLSTATE[08006] [7] connection to server at "db" (172.31.0.3), port 5432 fa  
  iled: FATAL:  the database system is starting up                             
                                                                               
doctrine:migrations:migrate [--write-sql [WRITE-SQL]] [--dry-run] [--query-time] [--allow-no-migration] [--all-or-nothing [ALL-OR-NOTHING]] [--no-all-or-nothing] [--configuration CONFIGURATION] [--em EM] [--conn CONN] [--] [<version>]
**** 1/11 - Make sure /uploads folders exist ****
**** 2/11 - Create the symbolic link for the /uploads folder ****
**** 3/11 - Setting env variables ****
**** 4/11 - Migrate the database ****
00:17:46 CRITICAL  [console] Error thrown while running command "'doctrine:migration:migrate' --no-interaction --allow-no-migration --env=prod". Message: "An exception occurred in the driver: SQLSTATE[08006] [7] connection to server at "db" (172.31.0.3), port 5432 failed: Connection refused\n\tIs the server running on that host and accepting TCP/IP connections?" ["exception" => Doctrine\DBAL\Exception\ConnectionException { …},"command" => "'doctrine:migration:migrate' --no-interaction --allow-no-migration --env=prod","message" => """  An exception occurred in the driver: SQLSTATE[08006] [7] connection to server at "db" (172.31.0.3), port 5432 failed: Connection refused\n  \tIs the server running on that host and accepting TCP/IP connections?  """]
In ExceptionConverter.php line 77:
                                                                               
  An exception occurred in the driver: SQLSTATE[08006] [7] connection to serv  
  er at "db" (172.31.0.3), port 5432 failed: Connection refused                
  	Is the server running on that host and accepting TCP/IP connections?         
                                                                               
In Exception.php line 28:
                                                                               
  SQLSTATE[08006] [7] connection to server at "db" (172.31.0.3), port 5432 fa  
  iled: Connection refused                                                     
  	Is the server running on that host and accepting TCP/IP connections?         
                                                                               
In PDOConnect.php line 26:
                                                                               
  SQLSTATE[08006] [7] connection to server at "db" (172.31.0.3), port 5432 fa  
  iled: Connection refused                                                     
  	Is the server running on that host and accepting TCP/IP connections?         
                                                                               
doctrine:migrations:migrate [--write-sql [WRITE-SQL]] [--dry-run] [--query-time] [--allow-no-migration] [--all-or-nothing [ALL-OR-NOTHING]] [--no-all-or-nothing] [--configuration CONFIGURATION] [--em EM] [--conn CONN] [--] [<version>]
**** 1/11 - Make sure /uploads folders exist ****
**** 2/11 - Create the symbolic link for the /uploads folder ****
**** 3/11 - Setting env variables ****
**** 4/11 - Migrate the database ****
00:18:24 CRITICAL  [console] Error thrown while running command "'doctrine:migration:migrate' --no-interaction --allow-no-migration --env=prod". Message: "An exception occurred in the driver: SQLSTATE[08006] [7] could not translate host name "db" to address: Name or service not known" ["exception" => Doctrine\DBAL\Exception\ConnectionException { …},"command" => "'doctrine:migration:migrate' --no-interaction --allow-no-migration --env=prod","message" => "An exception occurred in the driver: SQLSTATE[08006] [7] could not translate host name "db" to address: Name or service not known"]
In ExceptionConverter.php line 77:
                                                                               
  An exception occurred in the driver: SQLSTATE[08006] [7] could not translat  
  e host name "db" to address: Name or service not known                       
                                                                               
In Exception.php line 28:
                                                                               
  SQLSTATE[08006] [7] could not translate host name "db" to address: Name or   
  service not known                                                            
                                                                               
In PDOConnect.php line 26:
                                                                               
  SQLSTATE[08006] [7] could not translate host name "db" to address: Name or   
  service not known                                                            
                                                                               
doctrine:migrations:migrate [--write-sql [WRITE-SQL]] [--dry-run] [--query-time] [--allow-no-migration] [--all-or-nothing [ALL-OR-NOTHING]] [--no-all-or-nothing] [--configuration CONFIGURATION] [--em EM] [--conn CONN] [--] [<version>]
**** 1/11 - Make sure /uploads folders exist ****
**** 2/11 - Create the symbolic link for the /uploads folder ****
**** 3/11 - Setting env variables ****
**** 4/11 - Migrate the database ****
00:18:55 CRITICAL  [console] Error thrown while running command "'doctrine:migration:migrate' --no-interaction --allow-no-migration --env=prod". Message: "An exception occurred in the driver: SQLSTATE[08006] [7] could not translate host name "db" to address: Name or service not known" ["exception" => Doctrine\DBAL\Exception\ConnectionException { …},"command" => "'doctrine:migration:migrate' --no-interaction --allow-no-migration --env=prod","message" => "An exception occurred in the driver: SQLSTATE[08006] [7] could not translate host name "db" to address: Name or service not known"]
In ExceptionConverter.php line 77:
                                                                               
  An exception occurred in the driver: SQLSTATE[08006] [7] could not translat  
  e host name "db" to address: Name or service not known                       
                                                                               
In Exception.php line 28:
                                                                               
  SQLSTATE[08006] [7] could not translate host name "db" to address: Name or   
  service not known                                                            
                                                                               
In PDOConnect.php line 26:
                                                                               
  SQLSTATE[08006] [7] could not translate host name "db" to address: Name or   
  service not known                                                            
                                                                               
doctrine:migrations:migrate [--write-sql [WRITE-SQL]] [--dry-run] [--query-time] [--allow-no-migration] [--all-or-nothing [ALL-OR-NOTHING]] [--no-all-or-nothing] [--configuration CONFIGURATION] [--em EM] [--conn CONN] [--] [<version>]
